src/DynamicRupture/FrictionLaws/FrictionSolver.cpp

```diff
@@ -6,11 +6,14 @@ namespace seissol::dr::friction_law {
 
 void FrictionSolver::computeDeltaT(const double timePoints[CONVERGENCE_ORDER]) {
   deltaT[0] = timePoints[0];
+  sumDt = deltaT[0];
   for (unsigned timeIndex = 1; timeIndex < CONVERGENCE_ORDER; timeIndex++) {
     deltaT[timeIndex] = timePoints[timeIndex] - timePoints[timeIndex - 1];
+    sumDt += deltaT[timeIndex];
   }
   // to fill last segment of Gaussian integration
   deltaT[CONVERGENCE_ORDER - 1] = deltaT[CONVERGENCE_ORDER - 1] + deltaT[0];
+  sumDt += deltaT[CONVERGENCE_ORDER - 1];
```

Comment:
You add `deltaT[CONVERGENCE_ORDER - 1]` twice here. Is that intended?

Reply:
this was a mistake, that is now fixed. Thx!